### ISSUE TITLE
<audio> tag content messes up with the BBCode conversion process

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7665]
See: https://forum.friendi.ca/display/c443a55c-255d-8b60-3131-dff332345104

Provided input to `/babel`:
```html
Pour ceux qui ne l'aurait pas encore fait, je vous recommande <strong>vivement </strong>l'écoute de la conférence de <a href="http://jonathan-bocquet.fr/blog/" target="_blank" rel="nofollow noopener" >Jonathan Bocquet</a> sur le décentrement éthique.<br />
<audio src="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3" controls="controls" preload="none"><a href="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3</a></audio><br />
Il rappelle clairement que le #<a class="zrl" href="https://dacha.xmpp.moe/search?tag=JdR"  target="_blank"  rel="nofollow noopener" >JdR</a> est politique, puis il explique qu'un dilemme moral doit être diégétique : c'est le perso qui doit être tiraillé, pas le joueur.
```

BBCode output:
```
Pour ceux qui ne l'aurait pas encore fait, je vous recommande [b]vivement [/b]l'écoute de la conférence de [url=http://jonathan-bocquet.fr/blog/]Jonathan Bocquet[/url] sur le décentrement éthique.
[audio]http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3[url=http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3]http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3[/url][/audio]
Il rappelle clairement que le #[url=https://dacha.xmpp.moe/search?tag=JdR]JdR[/url] est politique, puis il explique qu'un dilemme moral doit être diégétique : c'est le perso qui doit être tiraillé, pas le joueur.
```

Input -> HTML to BBCode -> BBCode to HTML:
```html
Pour ceux qui ne l'aurait pas encore fait, je vous recommande <strong>vivement</strong> l'écoute de la conférence de <a href="http://jonathan-bocquet.fr/blog/" target="_blank">Jonathan Bocquet</a> sur le décentrement éthique.<br>
<audio src="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3<a%20href=" http: target="_blank">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3" controls="controls"><a href="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3<a%20href=" http: target="_blank">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3</a>"><a href="http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3<a%20href=" http: target="_blank">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3" target="_blank">http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3[url=http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3]http://www.cendrones.fr/colloque2017/jonathanbocquet.mp3</a></audio><br>
Il rappelle clairement que le #<a href="https://friendica.mrpetovan.com/search?tag=JdR" class="tag" title="JdR">JdR</a> est politique, puis il explique qu'un dilemme moral doit être diégétique : c'est le perso qui doit être tiraillé, pas le joueur.
```

**Comment by nupplaphil** [comment]
I added an important label too :X ;)

**Comment by nupplaphil** [comment]
Fixed by #7666
